inferentia/scripts/gen_triton_model.py

```diff
@@ -272,9 +272,8 @@ def get_tensorflow_initialize_impl():
 
         # TODO: Validate input/output from the model
 
-        # TODO: NEURONCORE_GROUP_SIZES is deprecated by AWS Neuron
         group_sizes = [str(1)] * cores_per_instance
-        os.environ["NEURONCORE_GROUP_SIZES"] = ','.join(group_sizes)
+        os.environ["NEURON_RT_NUM_CORES"] = ','.join(group_sizes)
```

Comment:
Since the [documentation](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/neuron-guide/neuron-runtime/nrt-configurable-parameters.html#using-neuron-rt-num-cores) mentions the `NEURON_RT_VISIBLE_CORES` is more specific and will take precedence over `NEURON_RT_NUM_CORES`, so switching to  `NEURON_RT_VISIBLE_CORES` instead.